src/wallet/wallet.cpp

```diff
@@ -2945,12 +2949,15 @@ bool CWallet::AddToWalletIfInvolvingMe(
         }
 
         // Orchard
-        mapOrchardActionData_t orchardActionData;
+        std::optional<OrchardWalletTxMeta> orchardTxMeta;
         if (consensus.NetworkUpgradeActive(nHeight, Consensus::UPGRADE_NU5)) {
-            orchardActionData = orchardWallet.AddNotesIfInvolvingMe(tx);
+            orchardTxMeta = orchardWallet.AddNotesIfInvolvingMe(tx);
         }
 
-        if (fExisted || IsMine(tx) || IsFromMe(tx) || sproutNoteData.size() > 0 || saplingNoteData.size() > 0 || orchardWallet.TxContainsMyNotes(tx.GetHash()))
```

Comment:
With the change to this line, do we use `OrchardWallet::TxInvolvesMyNotes` anywhere now? No harm in leaving it around I guess...